src/components/home.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaSearch } from 'react-icons/fa';
+import { fetchCoins } from '../redux/coin/coin';
+import Display from './display';
+
+const Home = () => {
+  const dispatch = useDispatch();
+
+  const { coins, status } = useSelector((state) => state.coins);
+  const [search, setSearch] = useState('');
+  useEffect(() => {
+    if (status === null) {
+      dispatch(fetchCoins());
+    }
+  }, [dispatch]);
+
+  return (
+    <div>
+      {status === 'pending' ? (
+        <div className="pending">
+          <p className="round" />
+          <h3>LOADING...</h3>
+        </div>
+      ) : (
+        <>
+          <input
+            type="text"
+            value={search}
+            onChange={(e) => setSearch(e.target.value)}
+            placeholder="search"
+          />
+          <FaSearch />
+
+          <div className="flex jstCnt flxWrp aliCnt ">
+            {coins.filter((searchCoin) => searchCoin.name.toLowerCase()
+              .includes(search.toLowerCase())
+              || searchCoin.symbol.toLowerCase().includes(search.toLowerCase()))
+              .map((coins) => (
+                <div className="flxCnt flex jstCnt aliCnt flxWrp gap10px sec1" key={coins.id}>
+                  <NavLink className="coinLink" state={coins} to="/display">
+                    <Display
+                      key={coins.id}
+                      img={coins.image}
+                      id={coins.id.toUpperCase()}
+                      name={coins.name}
+                      symbol={coins.symbol.toUpperCase()}
+                      atl={coins.atl}
+                      price={coins.current_price}
+                    />
+                  </NavLink>
+                </div>
+              ))}
+          </div>
+        </>
+      )}
+    </div>
+  );
+};
```

Comment:
- Good job implementing all of the features. However, most of what's left is to properly adjust your design to align more with the original design given by Microverse. Here are the changes you can focus on to do so:

   - [ ] Adjusting the colors of the cards and background to properly match the given design.
   - [ ] Adjusting the positioning of the header title to be more centered.
   - [ ] Changing the "X" button to another icon that represents a back button and aligning it to be on the left of the navbar to match the design.
   - [ ] Adding more spacing around the search bar so that it doesn't look like it's stuck to the left of the page.
   - [ ] Adding some inner padding in the displayed cards to match the design.
   - [ ] Using a similar font family and color to the one used in the original design.
   
(Kindly note that all of these changes need to be done in both the home and details pages since both pages are similar in their styles)

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/204554057-855a8521-47c9-4095-9bd3-d2758b46812c.png) | ![image](https://user-images.githubusercontent.com/62290545/204554098-c12a0057-08b9-46f2-a990-595a3ea691ea.png) |

Reply:
![Screenshot_20221129-163232_GitHub.png](https://user-images.githubusercontent.com/109461921/204574196-0063b42a-bd1e-4859-8ccf-c707a469f7c4.png)

As seen in the screenshot I attached, we are not restricted to follow the colors used in the template

Reply:
Except that, I'll properly style the application